tests/compile.sh

```diff
@@ -54,6 +54,7 @@ BUILD_JOBS=${BUILD_JOBS:-8}
 
 hostname
 
+module purge
```

Comment:
No, this must be in machine specific files. If you do `module purge` on gaea, for example, the whole system falls apart.

Reply:
This is still an issue @SMoorthi-emc we can't module purge on all machines, it breaks Gaea for example.